charts/spire/README.md

```diff
@@ -332,6 +332,10 @@ Now you can interact with the Spire agent socket from your own application. The
 | spire-agent.readinessProbe.initialDelaySeconds | int | `15` | Initial delay seconds for readinessProbe |
 | spire-agent.readinessProbe.periodSeconds | int | `60` | Period seconds for readinessProbe |
 | spire-agent.resources | object | `{}` |  |
+| spire-agent.sds.default_all_bundles_name | string | `"ALL"` | The Validation Context resource name to use for all bundles (including federated) with Envoy SDS |
+| spire-agent.sds.default_bundle_name | string | `"ROOTCA"` | The Validation Context resource name to use for the default X.509 bundle with Envoy SDS |
+| spire-agent.sds.default_svid_name | string | `"default"` | The TLS Certificate resource name to use for the default X509-SVID with Envoy SDS |
+| spire-agent.sds.disable_spiffe_cert_validation | bool | `false` | Disable Envoy SDS custom validation |
```

Comment:
These should be updated to follow the convention of the chart. See this for details:
https://github.com/spiffe/helm-charts/blob/main/project/conventions.md#preferred-characters-for-helm-path-elements

Reply:
@kfox1111 , are you referring to the preference for camel case?

Reply:
I think this is addressed